src/route/userActivity.route.ts

```diff
@@ -212,6 +213,26 @@ const userActivityRouter = () => {
     }
   );
 
+  router.post(
+    '/activities/upload-image',
```

Comment:
```suggestion
  router.post(
    '/upload-image',
```
move to general/user route file since it's used for all image upload

Reply:
updated